migtests/tests/oracle/utils/delete_full_schema.sql

```diff
@@ -1,18 +1,22 @@
+-- TODO: Enhance the script to include the deletion of dependent objects.
 SET ECHO ON
 
 BEGIN
    FOR cur_rec IN (SELECT object_name, object_type
                      FROM user_objects
                     WHERE object_type IN
                              ('TABLE',
+                              'TYPE',
+                              'TABLE PARTITION',
                               'VIEW',
                               'PACKAGE',
                               'PROCEDURE',
                               'FUNCTION',
                               'SEQUENCE',
-                              'TYPE',
                               'SYNONYM',
-                              'MATERIALIZED VIEW'
+                              'INDEX',
```

Comment:
there is something like `INDEX PARTITION` type also in oracle, which you can add.